src/save.cpp

```diff
@@ -548,12 +544,12 @@ void fread_char(Char *ch, FILE *fp) {
                 int sn;
                 sn = skill_lookup(fread_word(fp));
                 if (sn < 0)
-                    bug("Fread_char: unknown skill.");
+                    bug("fread_char: unknown skill.");
                 else
                     af.type = sn;
             } else /* old form */
                 af.type = fread_number(fp);
-            if (ch->version == 0)
+            if (ch->version == CharVersion::Zero)
```

Comment:
Is this even possible now? Seems like not?

Reply:
Yeah I think so, killing